packs/data/equipment.db/heartripper-blade.json

```diff
@@ -0,0 +1,143 @@
+{
+    "_id": "bcZB7xmlQy8UWqe1",
+    "img": "systems/pf2e/icons/default-icons/weapon.svg",
+    "name": "Heartripper Blade",
+    "system": {
+        "MAP": {
+            "value": ""
+        },
+        "baseItem": "dagger",
+        "bonus": {
+            "value": 0
+        },
+        "bonusDamage": {
+            "value": 0
+        },
+        "category": "simple",
+        "containerId": null,
+        "damage": {
+            "damageType": "piercing",
+            "dice": 1,
+            "die": "d4",
+            "value": ""
+        },
+        "description": {
+            "value": "<p>This wicked-looking curved weapon is a <em>+1 striking @UUID[Compendium.pf2e.equipment-srd.rQWaJhI5Bko5x14Z]{Dagger} </em>that draws power from defeating foes, either to bolster itself or its wielder. A heartripper blade is well suited for use in any ritual that involves divination, and incorporating it into any ritual with the divination trait grants the wielder a +1 item bonus on any check made to resolve the effects of the ritual's casting.</p>\n<hr />\n<p><strong>Activate</strong><span class=\"pf2-icon\">R</span> command</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> You reduce a foe to 0 Hit Points with a Strike from the <em>heartripper blade</em></p>\n<p><strong>Effect</strong> The <em>heartripper blade</em> draws power from the devastating strike, causing its blade to erupt in black flames. For 1 minute, Strikes with the <em>heartripper blade</em> inflict an additional [[/r {1d6}[negative]]]{1d6 negative damage} to living creatures.</p>\n<hr />\n<p><strong>Activate</strong><span class=\"pf2-icon\">R</span> command</p>\n<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> You reduce a foe to 0 Hit Points with a Strike from the heartripper blade</p>\n<p><strong>Effect</strong> The heartripper blade draws life from the victim's body and siphons it into you. Choose one of the following effects: restore [[/r {2d8}]]{2d8 Hit Points}; reduce the value of your clumsy, enfeebled, or stupefied conditions by 1; or reduce the stage of one toxin or disease you're suffering by one stage (this can't reduce the stage below stage 1 or cure the affliction).</p>"
```

Comment:
`{Dagger} </em>that` fix the spacing.

Make the inline healing `[[/r {2d8}[Healing]]]{2d8 Hit Points}`

Make an effect for the first reaction.

Link the conditions in the last reaction.